pkgs/development/libraries/ta-lib/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, autoPatchelfHook, pkg-config
+, gcc }:
+
+stdenv.mkDerivation rec {
+  name = "ta-lib";
+  version = "0.4.0";
+  src = fetchFromGitHub {
+    owner = "rafa-dot-el";
+    repo = "talib";
+    rev = "${version}";
+    sha256 = "sha256-bIzN8f9ZiOLaVzGAXcZUHUh/v9z1U+zY+MnyjJr1lSw=";
+  };
+
+  nativeBuildInputs = [ gcc pkg-config autoreconfHook autoPatchelfHook ];
```

Comment:
Are `gcc` and `autoPatchelfHook` really required?

`gcc` should be part of `stdenv` already.

`autoPatchelfHook` shouldn't be needed when building something from source.

Reply:
I removed then and it builds correctly, thanks for pointing it out!